tests/online_ddl/run.sh

```diff
@@ -33,15 +33,35 @@ function real_run() {
     sed -i "s/online-ddl-scheme-placeholder/${online_ddl_scheme}/g" $WORK_DIR/dm-task-${online_ddl_scheme}.yaml
     dmctl_start_task "$WORK_DIR/dm-task-${online_ddl_scheme}.yaml"
 
-    # use sync_diff_inspector to check full dump loader
+    echo "use sync_diff_inspector to check full dump data"
     check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
-    # skip online ddl test
-    # run_sql_file_online_ddl $cur/data/db1.increment.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1 online_ddl $online_ddl_scheme 
-    # run_sql_file_online_ddl $cur/data/db2.increment.sql $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl $online_ddl_scheme
+    run_sql_file_online_ddl $cur/data/db1.increment.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1 online_ddl $online_ddl_scheme
+    run_sql_file_online_ddl $cur/data/db2.increment.sql $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl $online_ddl_scheme
 
-    # use sync_diff_inspector to check data now!
-    # check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
+    echo "use sync_diff_inspector to check increment data"
+    check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
+
+    echo "start dm-worker3 and kill dm-worker2"
+    ps aux | grep dm-worker2 |awk '{print $2}'|xargs kill || true
+    check_port_offline $WORKER2_PORT 20
+    rm -rf $WORK_DIR/worker2/relay_log
+
+    run_dm_worker $WORK_DIR/worker3 $WORKER3_PORT $cur/conf/dm-worker3.toml
+    check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER3_PORT
+
+    sleep 8
+    echo "wait and check task running"
+    run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
```

Comment:
seems already had retry, don't need to wait 8 seconds in L53?

Reply:
removed in https://github.com/pingcap/dm/pull/494/commits/4a5e461ef8ce96ed4d4c034b15b53a2a8bd48027.